timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -372,9 +378,6 @@ def _get_model_params(self) -> dict:
                 callbacks=self.callbacks,
             )
         )
-        # Support MXNet kwarg names for backwards compatibility
-        init_args.setdefault("lr", init_args.get("learning_rate", 1e-3))
-        init_args.setdefault("max_epochs", init_args.get("epochs", 100))
```

Comment:
Are we intentionally deprecating the backward-compatible argument names `learning_rate` and `epochs`?

If that's the case, maybe we should log a warning when unsupported arguments are present (so that we don't just quietly change the behavior for users who use the old args)?